akka-docs/rst/java/http/routing-dsl/directives/basic-directives/extractStrictEntity.rst

```diff
@@ -0,0 +1,21 @@
+.. _-extractStrictEntity-java-:
+
+extractStrictEntity
+===================
+
+Description
+-----------
+
+Extracts the strict http entity as ``HttpEntity.Strict`` from the :class:`RequestContext`.
+
+A timeout parameter is given and if the stream isn't completed after the timeout, the directive will be failed.
+
+.. warning::
+
+  The directive will read the entire request entity into memory regardless of size and effectively disable streaming.
```

Comment:
No, the http server puts a stage in the entity stream that will limit it by default. Unless you explicitly opt out of it with the `withoutSizeLimit` directive.